packages/gasket-helper-intl/docs/api.md

```diff
@@ -20,33 +20,29 @@ Name | Description
 [LocalesProps] | Props for a Next.js page containing locale and initial state
 [LocaleStatus] | Fetch status of a locale file
 
-
```

Comment:
This file is auto-generated with [this script](https://github.com/godaddy/gasket/blob/docs-revamp/packages/gasket-helper-intl/package.json#L20).